python/classify.py

```diff
@@ -104,15 +104,20 @@ def main(argv):
 
     if args.gpu:
         print('GPU mode')
+    else:
+        print 'CPU mode'
```

Comment:
Right -- thanks for the reminder. This was just a cherry-pick so I'm still making changes and combing over.